context/app/static/js/components/Home/BarChart/BarChart.jsx

```diff
@@ -131,6 +107,18 @@ function BarChart(props) {
     setStyle({ maxHeight: 790, gridArea: 'bar', width: chartWidth + 38 }); // 38 = padding for vega button
   }, [windowDimensions]);
 
+  const labelAngle = useMediaQuery('(min-width:1150px)') ? '-50' : '-65';
+
+  useEffect(
+    () =>
+      setSpec((prevState) => {
+        const temp = prevState.encoding;
+        temp.x.axis.labelAngle = labelAngle;
+        return { ...prevState, ...temp };
+      }),
+    [labelAngle],
+  );
```

Comment:
I'd prefer to leave this out, and just be more conservative for all screen sizes, but ... never mind, we talked about this in the call.